src/index.ts

```diff
@@ -1,8 +1,8 @@
-export { ConnectionConfig, ConnectionConfigProps } from './ConnectionConfig';
+export { ConnectionConfig, type ConnectionConfigProps } from './ConnectionConfig';
```

Comment:
What does adding type here do?

Reply:
In Typescript, types are erased because js doesn't actually have the same runtime concept of types like ts. But when code is being transpiled, this export statement is actually trying to find something exported from the ConnectionConfig module, but since it's erased, it doesn't actually find anything. Adding type here indicates that it's just importing a type and there's nothing actually exported.